content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -39,31 +39,37 @@ model where `Pods` can be treated much like VMs or physical hosts from the
 perspectives of port allocation, naming, service discovery, load balancing,
 application configuration, and migration.
 
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (which is an optional capability). Even if a host have
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, so the "IP-per-pod" model is guaranteed.
+
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * pod-network pods on a node can communicate with all pod-network pods on
+     all nodes without NAT
```

Comment:
How about:
```suggestion
  * Any pod can try to send traffic towards any Pod that is also running with
    `hostNetwork: false`. The network addressing scheme must
    allow this.
    Security / filtering mechanisms such as NetworkPolicy could mean that a particular
    pod-to-pod packet is dropped or rejected.
```
This ties in with the rewritten note about SNAT possibly being needed for hostNetwork Pods.

Reply:
despite the correctness of the suggestion, I found the previous sentence easier to understand.
I've also found it in multiple slides and presentation. 
I think that at this point the `pods on a node can communicate with all pods on all nodes without NAT` is kind of the "1st commandment of the networking model"

Reply:
I see what @sftim is going for, but I think I agree with @aojea.  This is a pretty technical doc and I find the precision and brevity valuable.  

Does it make it better or worse to be more explicit?

* Any pod-network pod on any node can communicate with all other pod-network pods on all nodes without NAT.

Reply:
> Any pod-network pod on any node can communicate with all other pod-network pods on all nodes without NAT.

Sounds good to me. It'd be good to clarify somewhere in the page that being able to address the other Pod doesn't imply that your packets actually arrive there.

Reply:
is not covered by this sentence?
> Kubernetes imposes the following fundamental requirements on any networking implementation (barring any intentional network segmentation policies):